### PR TITLE
A0-1763: Remove v0 connections from clique

### PR DESCRIPTION
**Pull request by krzysztofziobro**
# Description

Removing unidirectional connections along with other "legacy" code

## Type of change

Please delete options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: finality-aleph/src/network/clique/protocols/v1/mod.rs [hunk 1]
```diff
@@ -28,6 +27,19 @@ enum Message<D: Data> {
     Heartbeat,
 }
 
+async fn check_authorization<SK: SecretKey>(
```
Comment (timorl) [inline finality-aleph/src/network/clique/protocols/v1/mod.rs:30]: Huh, what? This is part of @fixxxedpoint 's task, right? Why do I see it in this diff?

**Review by timorl — APPROVED** [review]
-1k lines, lets go!

File: finality-aleph/src/network/clique/protocols/v1/mod.rs (hunk 1 above)
Comment (fixxxedpoint) [reply finality-aleph/src/network/clique/protocols/v1/mod.rs:30]: moved from `v0/mod.rs`

**Review by fixxxedpoint — APPROVED** [review]
lgtm

File: finality-aleph/src/network/clique/protocols/v1/mod.rs (hunk 1 above)
Comment (timorl) [reply finality-aleph/src/network/clique/protocols/v1/mod.rs:30]: Ah...

**Pull request merged**